### ISSUE TITLE
Does not work with NVIM HEAD ( v0.10.0-dev-2571+gac8cd5368-Homebrew )

### ISSUE THREAD

**Issue by sbyang** [Skosulor/nibbler#2]
Thanks for the cool plugin.   Having problem getting it to work with my neovim setup.   Not sure if it is because of LazyVim, or the HEAD version of neovim.

NVIM v0.10.0-dev-2571+gac8cd5368-Homebrew
Build type: Release
LuaJIT 2.1.1703358377
Run "nvim -V1 -v" for more info

```
Error  01:46:15 PM msg_show.lua_error   NibblerToDec Error executing Lua callback: ...vim/HEAD-ac8cd53/share/nvim/runtime/lua/vim/lsp/util.lua:423: Explicit buffer number is required
stack traceback:
	[C]: in function 'assert'
	...vim/HEAD-ac8cd53/share/nvim/runtime/lua/vim/lsp/util.lua:423: in function 'apply_text_edits'
	...ang/.local/share/nvim/lazy/nibbler/lua/nibbler/edits.lua:68: in function 'replace_range_with'
	...ang/.local/share/nvim/lazy/nibbler/lua/nibbler/edits.lua:76: in function 'replace_word_under_cursor_with'
	...yang/.local/share/nvim/lazy/nibbler/lua/nibbler/init.lua:108: in function 'convert_selected_base'
	...yang/.local/share/nvim/lazy/nibbler/lua/nibbler/init.lua:213: in function <...yang/.local/share/nvim/lazy/nibbler/lua/nibbler/init.lua:213>

```

**Comment by sbyang** [comment]
For a workaround, nibbler worked after removing following line (assert). 
/home/linuxbrew/.linuxbrew/Cellar/neovim/HEAD-ac8cd53/share/nvim/runtime/lua/vim/lsp/util.lua
![image](https://github.com/Skosulor/nibbler/assets/5035711/2429e5ce-05ea-4b97-b1f0-a55dacfeb4ed)

**Comment by PriceHiller** [comment]
I just realized that this is probably solved with the merging of #3.

**Comment by sbyang** [comment]
Yup,  Thank you!
